lib/ftu/signuplogin.dart

```diff
@@ -0,0 +1,248 @@
+import 'package:flutter/material.dart';
+
+import '../constants/signuplogin_constants.dart';
+import './authentication.dart';
+
+// Changing from stateless to stateful
```

Comment:
Please delete all comments from this file.

Reply:
GREAT